### ISSUE TITLE
TypeError: Cannot read property 'syntax' of undefined

### ISSUE THREAD

**Issue by jitendravyas** [stylelint/stylelint#2629]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Not sure if it's a Bug or something else

> Which rule, if any, is this issue related to?

Not specific rule I guess but I tried with `--fix` with CLI and my stylelint has this rule only `"color-hex-length": "short",`

> What CSS is needed to reproduce this issue?

I'm not sure. I have lots of css in lots of files

> What stylelint configuration is needed to reproduce this issue?

```
{

	"rules": {
		"color-hex-length": "short"

	}
}
```

> Which version of stylelint are you using?

7.11.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`stylelint "app/styles/**/*.scss" --fix` and I assume it will automatically detect my `.stylelintrc` which is in same folder where I'm running the command

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Not sure

> What actually happened (e.g. what warnings or errors you are getting)?

```
TypeError: Cannot read property 'syntax' of undefined
    at stylelint._lintSource.then.postcssResult (/Users/jitendravyas/.nvm/versions/node/v6.10.0/lib/node_modules/stylelint/lib/standalone.js:156:74)

```

**Comment by hudochenkov** [comment]
Sorry you had to recreate this issue, but it was reported using template 2 hours ago. 

Duplicates #2627.
